### ISSUE TITLE
coursera-dl.conf is not working

### ISSUE THREAD

**Issue by adiah80** [coursera-dl/coursera-dl#686]
🚨Please review the [Troubleshooting](../#troubleshooting) section
before reporting any issue. Don't forget to check also the current issues to
avoid duplicates.

### Subject of the issue
I am unable to use coursera-dl.conf. 

### Your environment
* Operating System (name/version): Windows 10
* Python version: 3.63
* coursera-dl version:  0.11.4

### Steps to reproduce
I am using the following command : 
`coursera-dl competitive-data-science`

I have stored the coursera-dl.conf file in the same folder in which the script is executing. These are the contents of the file:
`--username <email> --password <password>`

However, I am not able to download the course and get the following error : 
`Please provide a username with the -u option, or a .netrc file with the -n option.`

The script works perfectly when I skip the conf file and put the username and password in the command itself : 
`coursera-dl --username <email> --password <password> competitive-data-science`

I have also tried the following commands : 
`-u <email> -p <password>`
`--username "<email>" --password "<password>"`
`-u "<email>" -p "<password>"`

Here is the error message with the --debug command included:
`root[parse_args] Please provide a username with the -u option, or a .netrc file with the -n option.`

* Is the problem happening with the latest version of the script?
* Do you have all the recommended versions of the modules? See them in the
  file `requirements.txt`.
* What is the course that you are trying to access?
* What is the precise command line that you are using (don't forget to obfuscate
  your username and password, but leave all other information untouched).
* What are the precise messages that you get? Please, use the `--debug`
  option before posting the messages as a bug report. Please, copy and paste
  them.  Don't reword/paraphrase the messages.

### Expected behaviour
Tell us what should happen.

### Actual behaviour
Tell us what happens instead. If the script fails, please copy the *entire*
output of the command or the stacktrace (don't forget to obfuscate your
username and password). If you cannot copy the exception, attach a screenshot.

**Comment by balta2ar** [comment]
Try this: https://github.com/coursera-dl/coursera-dl/issues/668#issuecomment-396551379

**Comment by adiah80** [comment]
I tried all three options, 
```
username = EMAIL 
password = PASSWORD
```
```
--username EMAIL
--password PASSWORD
```

as well as,
```
print(conf_file_path)
print(open(conf_file_path).read())
```

None of them seem to work. They all give the same error message as before.

**Comment by balta2ar** [comment]
Are you sure you're modifying and running the same file? See my last comment: https://github.com/coursera-dl/coursera-dl/issues/668#issuecomment-396610576

**Comment by adiah80** [comment]
Yeah coursera-dl.conf is in the current working directory. I can see it in the list of sub-directories/files when i run the `dir` command.

**Comment by adiah80** [comment]
Yeah, i tried putting it in commandline.py after the 44th line : 
```
conf_file_path = os.path.join(os.getcwd(), LOCAL_CONF_FILE_NAME)
    if os.path.isfile(conf_file_path):
        parse_kwargs["default_config_files"] = [conf_file_path]
        print(conf_file_path)
        print(open(conf_file_path).read())
    parser = argparse.ArgParser(**parse_kwargs)
```

**Comment by balta2ar** [comment]
Aaand does it print the path to the config file and the config file itself?

**Comment by adiah80** [comment]
No, it doesn't.

**Comment by balta2ar** [comment]
Could you move the first `print` before the `if` as follows:
``` python
conf_file_path = os.path.join(os.getcwd(), LOCAL_CONF_FILE_NAME)
print(conf_file_path)
```

**Comment by adiah80** [comment]
I made same changes and now it shows this output : 
```
D:\[-] Downloads\[ Coursera-dl ]\coursera-dl.conf
--username <email> --password <password>
Please provide a username with the -u option, or a .netrc file with the -n option.
```
when i enter : 
```
coursera-dl competitive-data-science
```

I had saved the file as 'coursera-fl.conf.txt'. I changed that to 'coursera-dl.conf'. 

After doing this : 
```
conf_file_path = os.path.join(os.getcwd(), LOCAL_CONF_FILE_NAME)
print(conf_file_path)
``` 
I get this : 
`D:\[-] Downloads\[ Coursera-dl ]\coursera-dl.conf`

**Comment by balta2ar** [comment]
Could you repeat the experiment in another folder which doesn't contain these funny characters `[-]` and is lower case, for example: `D:\downloads\coursera\`.

Also let's stay with the following coursera-dl.conf format -- one option per line:
```
username = EMAIL 
password = PASSWORD
```

**Comment by adiah80** [comment]
Finally, it works. 

Any idea how I can get it to work in the previous folder?

**Comment by balta2ar** [comment]
I've just googled my ass off and I haven't found a way... There is pywin32 with CreateFile, but I can't install it from pip and I'm not even sure it will help... That was just an idea to try.

If you are aware of any method, please let me know. I don't understand why Windows has be such a pain in the... back.

**Comment by adiah80** [comment]
Okay, I'll look around and let you know if I can find something.

Thanks for all the help!

**Comment by andrewng12** [comment]
u may try this, it's easy to setup and use
https://github.com/bcbcbcbcbcl/working-coursera-download-with-selenium

**Comment by IbraamYounan** [comment]
please, .conf file should be beside coursera-dl.py in python sight package path or in the foldar we want to save downloads in? now i coppied it to sight package path in python foldar, but it not work even after added lines above, what shall i do? also script itself not work as reported by many users...
